sysbuild/Kconfig.secureboot

```diff
@@ -10,21 +10,253 @@ config SECURE_BOOT
 	  This option will be set if the first stage bootloader which verifies the signature of the
 	  application is enabled for one or multiple cores.
 
-config SECURE_BOOT_APPCORE
+menuconfig SECURE_BOOT_APPCORE
 	bool "Appcore"
 	select SECURE_BOOT
 	help
 	  Enable first stage bootloader for application core.
 
-config SECURE_BOOT_NETCORE
+if SECURE_BOOT_APPCORE
+
+config SECURE_BOOT_APPCORE_SUPPORTED_HASH_HARDWARE
+	bool
+	default y if SECURE_BOOT_HASH_TYPE_SHA256 && (SOC_SERIES_NRF91X || SOC_NRF52840)
```

Comment:
might be `...&& HAS_HW_NRF_CC310`

Reply:
No such Kconfig in a sysbuild context, there is no devicetree